libs/geospatial/src/main/java/org/codice/ddf/libs/geo/coordinate/DmsCoordinate.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.libs.geo.coordinate;
+
+import org.codice.ddf.libs.geo.GeoFormatException;
+
+/**
+ * Coordinate that stores and provides access to data using Degrees Minutes Seconds format
+ */
+public class DmsCoordinate {
+
+    private Latitude latitude = null;
+
+    private Longitude longitude = null;
+
+    /**
+     * Constructor which takes in a Latitude Object and Longitude Object
+     * <p>
+     * Note: an IllegalArgumentException will be thrown in the case the Latitude
+     * or Longitude objects are null.
+     *
+     * @param lat the Latitude value in the Decimal Degree Coordinate
+     * @param lon the longitude value in the Decimal Degree Coordinate
+     */
+    public DmsCoordinate(Latitude lat, Longitude lon) {
+        if (lat == null || lon == null) {
+            throw new IllegalArgumentException("Latitude and Longitude must be non null");
+        }
+        this.latitude = lat;
+        this.longitude = lon;
+    }
+
+    /**
+     * Constructor which converts a String in DMS format to a Latitude/Longitude paise.  The String format is
+     * DDMMSS[sss]HDDMMSS[sss]H where (latitude is first):<br/>
+     * <p>
+     * DD is Degrees from 00 to 90 (must always be 2 characters long) representing the latitude
+     * DDD is Degrees from 000 to 180 (must always be 3 characters long) representing longitude<br/>
+     * MM is Minutes from 00 to 59 (must always be 2 characters long)<br/>
+     * SS is Seconds from 00 to 59 (must always be 2 characters long)<br/>
+     * sss is optional centiseconds from 000 to 999 (must either not appear or be 3 characters long)<br/>
+     * H the North, South, East or West Hemisphere representing by the 'N', 'S;, 'E' or 'W' characters respectively
+     * <p>
+     * <p>
+     * Example of String without centiseconds: 224530N0550230E (approx 22.75 degrees north and 55.40 degrees east)<br/>
```

Comment:
Longitude conversion looks wrong. You are saying 55 02 30 E is 55.40, but 55.04 might be closer.